globus_automate_client/cli/auth.py

```diff
@@ -138,7 +142,7 @@ def load_tokens(self):
             )
 
     @staticmethod
-    def _make_jsonable(tokens: TokensInTokenCache) -> Dict[str, Any]:
+    def _make_jsonable(tokens) -> Dict[str, Any]:
```

Comment:
Same here: checking why annotation removed

Reply:
See answer above. You caught both lines from 2f7bf211d6320fa05f3b64b1d06e2a01e40aacd4 that are, to sum it up, incorrect fixes.